main.py

```
@@ -0,0 +1,179 @@
+        status_code=500, detail=f"Error al inicializar Vertex AI: {str(e)}"
+    )
+
+# Inicializa el modelo Gemini
+model = GenerativeModel("gemini-2.5-flash-lite")
+
+# Inicializa el cliente de Vertex AI Search (Discovery Engine)
+# El cliente usará automáticamente las credenciales configuradas por vertexai.init
+# o por GOOGLE_APPLICATION_CREDENTIALS.
+try:
+    search_client = discoveryengine.SearchServiceClient()
+except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)